src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -30,14 +36,17 @@
 
 enum DriveStates {
     FIELD_ABSOLUTE,
-    FIELD_RELATIVE
+    FIELD_RELATIVE,
+    TELEOP_ALIGNING
 }
 
 public class Drive extends SubsystemBase {
     SwerveDrive swerveDrive;
     DriveStates driveStates = DriveStates.FIELD_RELATIVE;
     Robot robot = null;
     boolean fieldRelative = false;
+    Pose2d currentPose = new Pose2d(0, 0, new Rotation2d(0, 0));
```

Comment:
No reason for this to be a class variable -- you're already getting it in each local method, just make it a local variable in both cases